cloudinit/distros/networking.py

```diff
@@ -103,7 +111,8 @@ def wait_for_physdevs(
 class BSDNetworking(Networking):
     """Implementation of networking functionality shared across BSDs."""
 
-    pass
+    def is_physical(self, devname: DeviceName) -> bool:
+        raise NotImplementedError()
```

Comment:
So then our expectation related to the bug is that we are closing [LP: #1884619](http://pad.lv/1884619)  as that bug is really a refactor-only branch, not an add feature support parity on distroX right?

Reply:
Yes.